test/SlowTests/Issues/RavenDB_11906.cs

```diff
@@ -129,7 +129,6 @@ public void SupportForCreateFieldWithOptions(Options options)
 
         [Theory]
         [RavenData(SearchEngineMode = RavenSearchEngineMode.Lucene)]
-        [RavenData(SearchEngineMode = RavenSearchEngineMode.Corax, Skip = "RavenDB-17966")]
```

Comment:
Shouldn't we also set `SearchEngineMode = RavenSearchEngineMode.All` now?

Reply:
Okay. So we can update the issue number here - dynamic fields is "RavenDB-18144"